compiler/modules/print.wasm/tests/printBoolean.bal

```diff
@@ -0,0 +1,26 @@
+import ballerina/test
+@test:Config {}
+
+public function printBoolean() returns error? {
+    Module module = new;
+    Type[] localTypes = ["i32"];
+    module.addFunctionImport("println", "console", "log", "i32", "None");
+    Expression b = module.localGet(0, "i32");
+    Expression[] thenCallOperands = [module.addConst(<LiteralInt32>{i32: 1})];
+    Expression thenCall = module.call("println", thenCallOperands, 1, "None");
+    Expression[] elseCallOperands = [module.addConst(<LiteralInt32>{i32: 0})];
+    Expression elseCall = module.call("println", elseCallOperands, 1, "None");
+    Relooper relooper = new(module);
```

Comment:
Did you look at C/C++ code and find out that Module has no references to Relooper?

Reply:
Yes module does not use the relooper object after creating.